cfgov/jinja2/v1/events/event.html

```diff
@@ -1,8 +1,8 @@
 {% extends 'layout-2-1-bleedbar.html' %}
 {% import '_vars-events.html' as vars with context %}
 {% import 'templates/streamfield-sidefoot.html' as streamfield_sidefoot with context %}
-{% if page.start_dt and page.end_dt %}
-    {% set event_state = when(page.start_dt | date('%c'), page.end_dt | date('%c')) %}
+{% if page.live_stream_date and page.end_dt %}
+    {% set event_state = when(page.live_stream_date, page.end_dt) %}
```

Comment:
Because the stream goes live before the event, and the only thing event_state is used for is checking if it should show the live stream during the event or the saved video after the event.